bin/GameIndex.dbf

```diff
@@ -8376,39 +8384,39 @@ Region = PAL-E
 Compat = 5
 vuRoundMode = 0
 vuClampMode = 3
-eeClampMode = 2	// fix missing/grey texture or geometry ingame
+eeClampMode = 2	// Fixes missing/grey texture or geometry ingame.
```

Comment:
Maybe the spacing on these comments could be adjusted.

There are others as well.